docs/quick-start/developers/lottery-game-smart-contract/index.md

```diff
@@ -1380,6 +1380,41 @@ Enter the required param <symbol>: ELF
 Enter the required param <owner>: $WALLET_ADDRESS
 ```
 
+## Understanding Inter-Contract Calls in aelf
+
+In this section, we'll explore how inter-contract calls work in the aelf blockchain using the lottery game example from the tutorial. This will help you understand how different smart contracts can interact to perform complex operations.
+
+### 1. **Smart Contract Overview**
+   - **Lottery Contract**: Manages the game, including buying tickets, drawing winners, and distributing prizes.
+   - **Token Contract**: Handles the token transactions needed for buying lottery tickets.
+
+### 2. **Ticket Purchase Process**
+   - **Initiating Purchase**: When a user wants to buy a lottery ticket, they interact with the Lottery Contract.
+   - **Token Transfer Requirement**: The Lottery Contract must verify that the user has enough tokens and transfer those tokens to the lottery’s account to complete the purchase.
+
+### 3. **Making an Inter-Contract Call**
```

Comment:
There needs to be a step before this regarding initialising contract reference state to the contract address. The rest are ok.